.github/workflows/gofmt.yml

```diff
@@ -0,0 +1,14 @@
+name: gofmt
+on: [push, pull_request]
+jobs:
+  gofmt:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 2
+    - run: git checkout HEAD^2
```

Comment:
Hm, out of curiosity, why are we reverting back with two commits here via `HEAD^2`?

Reply:
Hmm, I'd seen examples in the docs for actions, and figured it was a PR-specific issue. Can try it without.

Reply:
No worries, no blocking 👍